master/reference/advanced/etcd-rbac/kubernetes.md

```diff
@@ -2,12 +2,20 @@
 title: Setting up Kubernetes with Calico components utilizing etcd RBAC
 ---
 
-## Overview
+When using etcd with RBAC, all components that access etcd must be configured
+with the proper certificates.  This document describes the different
+components that need to be configured and offers some help with that
+configuration.
 
-When using etcd with RBAC all components that access etcd must be configured
-with the proper certificates.  Here are the different components that need to be
-configured and some help with that task for operating Kubernetes with Calico
-accessing an etcd with RBAC.
+## Why you might be interested in this guide
+
+- You are using Kubernetes with Calico, utilizing an etcd datastore, and want
+  to restrict the access that Calico components have in the etcd datastore.
+- You are using Kuberentes and Calico who share and etcd datastore and wish
```

Comment:
No, it is about K8s and Calico using a single etcd datastore instead of having a k8s dedicated one and a Calico dedicated one.